test/Integration/test-powershell/AddProductColumnTypes/run.ps1

```diff
@@ -10,9 +10,9 @@ Write-Host "PowerShell function with SQL Output Binding processed a request."
 # Note that this expects the body to be a JSON object or array of objects 
 # which have a property matching each of the columns in the table to upsert to.
 $req_query = @{ 
-    ProductId=$Request.QUERY.productId;
-    Datetime=Get-Date -AsUTC;
-    Datetime2=Get-Date -AsUTC;
+    ProductId=[int]$Request.QUERY.productId;
+    Datetime=Get-Date;
```

Comment:
Why are you removing AsUTC?

Reply:
Forgot to revert this for now. Will update.